app/main.py

```diff
@@ -1,6 +1,32 @@
+import os
+
+import requests
+
+
+URL = "https://api.weatherapi.com/v1/current.json?"
+
+FILTERING = "Ile-de-France"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    api_key = os.getenv("API_KEY")
+
+    if not api_key:
+        print("Error: API_KEY is not set in environment variables")
+        return
+
+    response = requests.get(
+        URL + "key=" + api_key + "&q=" + FILTERING + "&aqi=yes"
+    )
+    if response.status_code == 200:
+        data = response.json()
+        print(f"{data["location"]["name"]}/{data["location"]["country"]} "
+              f"{data["current"]["last_updated"]} "
+              f"Weather: {data["current"]["temp_c"]} Celsius, "
+              f"{data["current"]["condition"]["text"]}")
```

Comment:
The f-string uses double quotes inside the string, which will cause a syntax error. You should use single quotes inside the f-string or escape the double quotes. For example: `print(f"{data['location']['name']}/{data['location']['country']} ...")`.